scripts/run_ghopper.sh

```diff
@@ -3,9 +3,7 @@
 # grep -E -i ".*(\[main\]|\[INFO\]|erro?r?|exception|at.*(java|graphhopper)|exit*).*" # grep whitelist lines
 # grep -E -i "^((?!download|points?|maven).)*$" # grep blacklist lines
 
-cd docker-compose
 echo "Initializing Graphhopper Services..."
-docker-compose up --no-start --force-recreate ghopper # | grep -v -i 'points?0?\|download'
-docker-compose up | rotatelogs -f -l  -n 5 ./logs/.graphhopper.log 86400 1M
-cd ..
-echo "Exiting"
+docker-compose -f docker-compose/docker-compose.yml up --no-start --force-recreate ghopper # | grep -v -i 'points?0?\|download'
```

Comment:
can we delete the `# grep`?

Reply:
It's good to have in case you want filtered logs or console output